gpib_server.py

```diff
@@ -112,7 +112,7 @@ def refreshDevices(self):
                     elif addr.startswith('TCPIP'):
                         instName = addr
                     elif addr.startswith('USB'):
-                        instName = addr + '::INSTR'
+                        instName = addr
```

Comment:
Yeah, I was imagining that tuple as being a module-level global, just like you did it.